superdesk/vocabularies/vocabularies.py

```diff
@@ -164,6 +172,17 @@ def on_replaced(self, document, original):
         """
         self._send_notification(document)
 
+    def on_delete(self, doc):
+        """
+        Overriding to validate vocabulary deletion
+        """
+        if 'field_type' not in doc:
+            raise SuperdeskApiError.badRequestError('Default vocabularies cannot be deleted')
+
+    def delete(self, lookup):
```

Comment:
there is [soft delete](http://python-eve.org/features.html#soft-delete) we could use that